Formula/rpki-client.rb

```diff
@@ -34,6 +34,6 @@ def post_install
   end
 
   test do
-    assert_match "VRP Entries: 0 (0 unique)", shell_output("#{sbin}/rpki-client -n -d . -R . 2>&1").lines.last
+    assert_match "rpki-client-portable 8.0", shell_output("#{sbin}/rpki-client -V 2>&1").lines.last
```

Comment:
The previous test was failing because the order of the lines changed. We can keep both the previous and new test:
```ruby
assert_match "VRP Entries: 0 (0 unique)", shell_output("#{sbin}/rpki-client -n -d . -R . 2>&1").lines[-2]
assert_match "rpki-client-portable #{version}", shell_output("#{sbin}/rpki-client -V 2>&1").lines.last
```

Using `#{version}` is also better than hardcoding the version in the test, otherwise we would need to change at every version bump.

Reply:
You're right, here's a simpler version:
```ruby
assert_match "VRP Entries: 0 (0 unique)", shell_output("#{sbin}/rpki-client -n -d . -R . 2>&1")
assert_match "rpki-client-portable #{version}", shell_output("#{sbin}/rpki-client -V 2>&1")
```